jobserver/views/workspaces.py

```diff
@@ -353,14 +353,17 @@ def dispatch(self, request, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         return super().get_context_data(**kwargs) | {
+            "backends": Backend.objects.all(),
```

Comment:
Could we filter the backends so we only show those that have associated jobs in the workspace?  For instance if I was running something on TPP and EMIS, I wouldn't want to see Graphnet.

Reply:
Good point, done and rebased in.